src/compiler/factory.ts

```diff
@@ -93,12 +93,18 @@ namespace ts {
         return node;
     }
 
-    function createStringLiteral(text: string): StringLiteral {
+    export function createStringLiteral(text: string): StringLiteral {
         const node = <StringLiteral>createSynthesizedNode(SyntaxKind.StringLiteral);
         node.text = text;
         return node;
     }
 
+    export function createRegularExpressionLiteral(text: string): RegularExpressionLiteral {

```

Comment:
👍 no need to add unit test for this?

Reply:
there are no unit tests for the other factory functions